src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_ActiveFareProductsTabScreen.tsx

```diff
@@ -41,17 +54,53 @@ export const TicketTabNav_ActiveFareProductsTabScreen = () => {
             activeFareContractsCount: activeFareContracts.length,
           });
         }}
-        now={serverNow}
-        showTokenInfo={true}
-        emptyStateTitleText={t(
-          TicketingTexts.activeFareProductsAndReservationsTab
-            .noActiveTicketsTitle,
-        )}
-        emptyStateDetailsText={t(
-          TicketingTexts.activeFareProductsAndReservationsTab
-            .noActiveTicketsDetails,
-        )}
-      />
+      >
+        <FareContractAndReservationsList
+          reservations={reservations}
+          fareContracts={activeFareContracts}
+          now={serverNow}
+          showTokenInfo={true}
+          emptyStateTitleText={t(
+            TicketingTexts.activeFareProductsAndReservationsTab
+              .noActiveTicketsTitle,
+          )}
+          emptyStateDetailsText={t(
+            TicketingTexts.activeFareProductsAndReservationsTab
+              .noActiveTicketsDetails,
+          )}
+        />
+        <Section style={styles.content}>
+          {hasExpiredFareContracts && (
+            <LinkSectionItem
+              text={t(Ticketing.expiredTickets.title)}
+              accessibility={{
+                accessibilityHint: t(Ticketing.expiredTickets.a11yHint),
+              }}
+              testID="expiredTicketsButton"
+              onPress={() =>
+                navigation.navigate('Ticketing_TicketHistoryScreen', {
+                  mode: 'expired',
+                })
+              }
+            />
+          )}
+
+          {hasSentFareContracts && (
+            <LinkSectionItem
+              text={t(Ticketing.sentToOthers.title)}
+              accessibility={{
+                accessibilityHint: t(Ticketing.sentToOthers.a11yHint),
+              }}
+              testID="sentToOthersButton"
+              onPress={() =>
+                navigation.navigate('Ticketing_TicketHistoryScreen', {
+                  mode: 'sent',
+                })
+              }
+            />
+          )}
```

Comment:
For the future: Just skip adding boilerplate for future features. Those will be added in their resepective PR, and makes the current PR more concise.

Reply:
will keep that in mind!